scopes/component/component-descriptor/aspect-map.ts

```diff
@@ -0,0 +1,33 @@
+export type AspectDataEntry = {
+  aspectId: string;
+  aspectData: string;
+};
+
+export type AspectMapProps = {
+  entries?: AspectDataEntry[];
+};
+
+export class AspectMap {
```

Comment:
@odedre please delete this file.
We changed it's name, so this is just leftover in your env

Reply:
are you sure?

Reply:
@GiladShoham so just remove  this entire file?